packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.1.0
```

Comment:
This doesn't introduce new features so should be a patch bump (1.0.5)

Reply:
Done